docs/dev/agent_integration.md

```diff
@@ -21,6 +22,52 @@ The required Datadog Agent integration development tools include:
 - [Docker][4] to run the full test suite.
 - The git [command-line][5] or [GitHub desktop client][19].
 
+You can list an Agent integration for a price in the Datadog Marketplace, or list it out-of-the-box on the Integrations page.
+
+{{< tabs >}}
+{{% tab "Marketplace Integrations" %}}
+## Set up a directory and clone the Marketplace repository
+
+Set up a directory:
+
+1. Request access to the [Marketplace repository][101] by following the instructions in the [Marketplace documentation][102].
+2. Create a `dd` directory:
+   {{< code-block lang="shell" >}}mkdir $HOME/dd{{< /code-block >}}
+
+   The Datadog Development Toolkit command expects you to be working in the `$HOME/dd/` directory. This is not mandatory, but working in a different directory requires additional configuration steps.
+3. Once you have been granted access to the Marketplace repository, create the `dd` directory and clone the `marketplace` repo:
+   {{< code-block lang="shell" >}}git clone git@github.com:DataDog/marketplace.git{{< /code-block >}}
+4. Create a feature branch to work in.
+    {{< code-block lang="shell" >}}git switch -c <YOUR INTEGRATION NAME> origin/master{{< /code-block >}}
+
+## Install and configure the Datadog development toolkit
+
+The Agent Integration Developer Tool allows you to create scaffolding when you are developing an integration by generating a skeleton of your integration tile's assets and metadata. For instructions on installing the tool, see [Install the Datadog Agent Integration Developer Tool][103].
+
+After you install the Developer tool, configure it for the `marketplace` repo:
+
+Set `marketplace` as the default working repository:
+
+{{< code-block lang="shell" >}}
+ddev config set marketplace $HOME/dd/marketplace
+ddev config set repo marketplace
+{{< /code-block >}}
+
+If you used a directory other than `$HOME/dd` to clone the marketplace directory, use the following command to set your working repository:
+
+{{< code-block lang="shell" >}}
+ddev config set marketplace <PATH/TO/MARKETPLACE>
+ddev config set repo marketplace
+{{< /code-block >}}
+
+[101]: https://github.com/Datadog/marketplace
+[102]: https://docs.datadoghq.com/developers/integrations/marketplace_offering
+[103]: https://docs.datadoghq.com/developers/integrations/python
+
+{{% /tab %}}
+
+{{% tab "Out-of-the-box Integrations" %}}
+
 ## Set up your integrations-extra repo
```

Comment:
Is "Set up your" accurate here? Would "fork and clone the integrations-extras repository" or something similar be more specific? It's not their repo. They are just making a copy of the existing repo locally.

Reply:
We mention step by step in the instructions to fork and clone, I think including that at the title might lead partners to do that rather than following directions. This also keeps consistency with the marketplace instructions.

Reply:
I'm just calling out that, semantically, the user is not setting up their own integrations-extra repo, so I find the heading confusing. 

Same with the next line: "Follow these instructions to set up your repo for integration development"

Datadog owns the int-extras repo. We're only asking the user to set up their own _directory_ where they will fork and clone a copy of the int-extras repo